README.md

```diff
@@ -4,13 +4,13 @@
 [![Codacy Badge](https://img.shields.io/codacy/grade/f194807bd2fe48eabedef320abc91f0b)](https://www.codacy.com/manual/mateumann/clouseau?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=mateumann/clouseau&amp;utm_campaign=Badge_Grade)
 [![codecov](https://codecov.io/gh/mateumann/clouseau/branch/master/graph/badge.svg)](https://codecov.io/gh/mateumann/clouseau)
 [![license](https://img.shields.io/github/license/mateumann/clouseau)](https://github.com/mateumann/clouseau/blob/master/LICENSE)
-<!--[![Build](https://github.com/mateumann/clouseau/workflows/Build/badge.svg)](https://github.com/mateumann/clouseau/actions?query=workflow%3ABuild)
-[![Linting](https://github.com/mateumann/clouseau/workflows/Linting/badge.svg)](https://github.com/mateumann/clouseau/actions?query=workflow%3ALinting) -->
 
-Packet statistics collector (and analyser). The name is obviously inspired by the famous Inspector
-Clouseau and his adventures.
+Packet statistics collector (and analyser). The name is inspired by the famous
```

Comment:
Codacy found an issue: [[write-good] "is inspired" may be passive voice](https://app.codacy.com/gh/mateumann/clouseau/pullRequest?prid=6383124)